app/controllers/api/v1/rents_controller.rb

```diff
@@ -20,6 +20,7 @@ def create
       private
 
       def permitted_params
+        params[:user_id] = current_api_v1_user.id
```

Comment:
do we need this?

Reply:
I did this so the rent gets associated to the current user, because it doesn't make much sense to localize the email using current users' locale field if he isn't the owner of the rent.